README.md

```diff
@@ -1,5 +1,6 @@
 [![Build Status](https://travis-ci.org/SAP/python-pyodata.svg?branch=master)](https://travis-ci.org/SAP/python-pyodata)
 [![PyPI version](https://badge.fury.io/py/pyodata.svg)](https://badge.fury.io/py/pyodata)
+[![codecov](https://codecov.io/gh/SAP/python-pyodata/branch/master/graph/badge.svg)](https://codecov.io/gh/mamiksik/python-pyodata)
```

Comment:
Hmm, is the url "https://codecov.io/gh/mamiksik/python-pyodata" ok?

Reply:
We want the URL for SAP repo not mine fork, don't we?

Reply:
ahh I see my bad